src/connectors/listen.moe.js

```diff
@@ -3,7 +3,14 @@
 Connector.playerSelector = '#app';
 
 Connector.artistSelector = '.playerContainer.columns.mini .player .player-song-artist';
-
 Connector.trackSelector = '.playerContainer.columns.mini .player .player-song-title';
 
+Connector.getArtistTrack = () => {
+	return $('.player-song-title')[0].innerText.split(' [')[0];
```

Comment:
Can you give some examples of tracks which this helps?

Reply:
Ok... The page sometimes displays the Show (From the Series the Opening is from) and this is in this format: artist - Trackname [Some aditional show info that confuses lastfm]
and in the artist field if multiple groups did work on one track is weird too... let me search an example...

Here an nice example, where both the title and the Artists are bad: